
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Phylogeny &#8212; Comparative Animal Behavior</title>
    
  <link href="../_static/css/theme.css" rel="stylesheet">
  <link href="../_static/css/index.ff1ffe594081f20da1ef19478df9384b.css" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      

    
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-book-theme.css?digest=c3fdc42140077d1ad13ad2f1588a4309" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../_static/js/index.be7d3bbb2ef33a8344ce.js">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/sphinx-book-theme.d59cb220de22ca1c485ebbdc042f0030.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Evolution" href="evolution.html" />
    <link rel="prev" title="Analysis" href="../observation-and-analysis/analysis.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../_static/ABcover.jpg" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">Comparative Animal Behavior</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../intro.html">
   Welcome
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  General
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../syllabus.html">
   Syllabus
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../schedule.html">
   Schedule
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../assignments.html">
   Assignments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../genindex.html">
   Terms, Topics, Concepts
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../assessment-sum/collection.html">
   Summative Assessmenta
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_components.html">
     Assessment 1: Components of Behavior
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_observation-analysis.html">
     Assessment 2: Observation and Analysis of Behavior
    </a>
   </li>
  </ul>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Describing Behavior
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/components.html">
   Components
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/observation.html">
   Observation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/analysis.html">
   Analysis
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Levels of Analysis
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Phylogeny
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="evolution.html">
   Evolution
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="neural-hormonal-genetic.html">
   Neural, Hormonal, and Genetic
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="neural.html">
     Neural Mechanisms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="genetic.html">
     Genetic Mechanisms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="hormonal.html">
     Hormonal Mechanisms
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="development.html">
   Development
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Understanding Behavior
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/endogenous-rhythms.html">
   Endogenous Rhythms
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/navigation.html">
   Navigation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/escape.html">
   Escape
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../understanding-behavior/scales-of-sociality.html">
   Scales of Sociality
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/cooperation.html">
     Grouping and Cooperation
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/eusociality.html">
     Eusociality
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/communication.html">
   Communication
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../understanding-behavior/family-life.html">
   Family Life
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-4" name="toctree-checkbox-4" type="checkbox"/>
  <label for="toctree-checkbox-4">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/sexual-selection.html">
     Sexual Selection
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/mating-systems.html">
     Mating Systems
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/parental-care.html">
     Parental Care
    </a>
   </li>
  </ul>
 </li>
</ul>

    </div>
</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../_sources/levels-of-analysis/phylogeny.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
                onclick="printPdf(this)" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/neurologic/Comparative-Animal-Behavior"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        <a class="issues-button"
            href="https://github.com/neurologic/Comparative-Animal-Behavior/issues/new?title=Issue%20on%20page%20%2Flevels-of-analysis/phylogeny.html&body=Your%20issue%20content%20here."><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Open an issue"><i class="fas fa-lightbulb"></i>open issue</button></a>
        
    </div>
</div>

            <!-- Full screen (wrap in <a> to have style consistency -->

<a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
        data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
        title="Fullscreen mode"><i
            class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show noprint">
            
            <div class="tocsection onthispage pt-5 pb-3">
                <i class="fas fa-list"></i> Contents
            </div>
            <nav id="bd-toc-nav" aria-label="Page">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genetic-sequences">
   Genetic Sequences
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sequence-alignment">
   Sequence Alignment
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-relatedness-to-trees">
   From Relatedness to Trees
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-trees-to-behavioral-evolutionary-history">
   From Trees to Behavioral Evolutionary History
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#additional-resources">
   Additional Resources
  </a>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Phylogeny</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genetic-sequences">
   Genetic Sequences
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sequence-alignment">
   Sequence Alignment
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-relatedness-to-trees">
   From Relatedness to Trees
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-trees-to-behavioral-evolutionary-history">
   From Trees to Behavioral Evolutionary History
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#additional-resources">
   Additional Resources
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            
              <div>
                
  <div class="tex2jax_ignore mathjax_ignore section" id="phylogeny">
<h1>Phylogeny<a class="headerlink" href="#phylogeny" title="Permalink to this headline">¶</a></h1>
<hr>
<blockquote>
<div><p>1381 words | 55 min read</p>
</div></blockquote>
<hr>
<div class="admonition-why admonition" id="index-0">
<p class="admonition-title">Why?</p>
<p>Animal behaviorists use <strong>phylogenies</strong> to investigate the development of behavior across generations (evolutionary timescales).
Critically, a <strong>phylogeny</strong> is a hypothesis. Phylogenies can be tested and falsified, and can change given new evidence or analytic techniques.
Phylogenies can be based on any animal trait. Commonly, in studying animal behavior, you will encounter phylogenies based on genetics.</p>
</div>
<div class="section" id="genetic-sequences">
<h2>Genetic Sequences<a class="headerlink" href="#genetic-sequences" title="Permalink to this headline">¶</a></h2>
<p><a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/23193287">GenBank</a> is the NIH genetic sequence database, an annotated collection of all publicly available DNA sequences. The <a class="reference external" href="https://www.ncbi.nlm.nih.gov/nucleotide/">nucleotide search</a> lets you search for species names and genes. If you want to use this resource directly, you can refer to the <a class="reference external" href="https://www.ncbi.nlm.nih.gov/books/NBK44863/">FAQ</a> to get you started. In this document, I provide you with example sequences from the GenBank.</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 5 min</p>
</div></blockquote>
<p><em><strong>Q: What is the relationship between DNA and genes?</strong></em></p>
<p><em><strong>Q: What is the relationship between DNA and nucleotides?</strong></em></p>
<p><em><strong>Q: What is the relationship between DNA and RNA?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 5 min</p>
</div></blockquote>
<p><a id='fasta-files'></a>
Below are two <em>FASTA</em> files downloaded from GenBank. “FASTA” is a common file format used to store sequence data. The files are very simple; they contain a <em>header</em> followed by the sequence. Click on the file headers to see its contents.</p>
<details class="sphinx-bs dropdown card mb-3">
<summary class="summary-title card-header">
HQ377551.1 <em>Chelonia mydas</em> isolate Satang_4 16S ribosomal RNA gene, partial sequence; mitochondrial; length 510<div class="summary-down docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-down" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M5.22 8.72a.75.75 0 000 1.06l6.25 6.25a.75.75 0 001.06 0l6.25-6.25a.75.75 0 00-1.06-1.06L12 14.44 6.28 8.72a.75.75 0 00-1.06 0z"></path></svg></div>
<div class="summary-up docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-up" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M18.78 15.28a.75.75 0 000-1.06l-6.25-6.25a.75.75 0 00-1.06 0l-6.25 6.25a.75.75 0 101.06 1.06L12 9.56l5.72 5.72a.75.75 0 001.06 0z"></path></svg></div>
</summary><div class="summary-content card-body docutils">
<p class="card-text">GGCCGCGGTATCCTAACCGTGCAAAGGTAGCGTAATCACTTGTCTTTTAAATAAAGACTAGAATGAATGG<br />
CCAAACGAGGTTCTACCTGTCTCTTACAAACAATCAGTGAAATTGATCTCCCCGTGCAAAAGCGGGGATA
ACACTATAAGACGAGAAGACCCTGTGGAACTTTAAATACAGATCAACTATCATACCCACTCACTCTAAGG<br />
ACCTATAACTAATTAGTACTTGACCTATATTTTTGGTTGGGGCGACCTCGGAGTAAAACAAAACCTCCGA<br />
AAAAAGAATACACTTCTTAACCTAGACCCACAATTCAAAGTGCCAACGGCAAAATGATCCAATATATTTG<br />
ATCAACGAACCAAGCTACCCCAGGGATAACAGCGCAATCCCATCCTAGAGTTCCTATCGACGATGGGGTT<br />
TACGACCTCGATGTTGGATCAGGACATCCTGATGGTGCAACCGCTATCAAGGGTTCGTTTGTTCAACGAT<br />
TAACAGTCCTACGTGATCTG</p>
</div>
</details><details class="sphinx-bs dropdown card mb-3">
<summary class="summary-title card-header">
FJ039971.1 <em>Eretmochelys imbricata</em> isolate EI_Atlantic 16S ribosomal RNA gene, partial sequence; mitochondrial; length 560<div class="summary-down docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-down" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M5.22 8.72a.75.75 0 000 1.06l6.25 6.25a.75.75 0 001.06 0l6.25-6.25a.75.75 0 00-1.06-1.06L12 14.44 6.28 8.72a.75.75 0 00-1.06 0z"></path></svg></div>
<div class="summary-up docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-up" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M18.78 15.28a.75.75 0 000-1.06l-6.25-6.25a.75.75 0 00-1.06 0l-6.25 6.25a.75.75 0 101.06 1.06L12 9.56l5.72 5.72a.75.75 0 001.06 0z"></path></svg></div>
</summary><div class="summary-content card-body docutils">
<p class="card-text">GCCTCTAGCAACAACAAGTATTAGAGGTAATGCCTGCCCAGTGACACTGTTAAACGGCCGCGGTATCCTA
ACCGTGCAAAGGTAGCGTAATCACTTGTCTTTTAAATAAAGACTAGAATGAATGGCCAAACGAGGTTCCA
CCTGTCTCTTACAAACAATCAGTGAAATTGGTCTCCCCGTGCAAAAGCGAGGATAGCACTATAAGACGAG
AAGACCCTGTGGAACTTTAAATATAAATCAACTATTTAACTTACCACTCTAAAGACTTATAATTTACTAG
TTCTGATCCATATTTTTGGTTGGGGTGACCTCGGAGAAAAACAAAACCTCCGAAAAAAGAACATATCTTC
TTAACCTAGACCCACAACTCAAAGTGCCAACGGAAAAATGATCCAATATATTTGATCAACGAACCAAGCT
ACCCCAGGGATAACAGCGCAATCCCATCTTAGAGTCCATATCGACGATGGGGTTTACGACCTCGATGTTG
GATCAGGACATCCTGATGGTGCAACCGCTATCAAGGGTTCGTTTGTTCAACGATTAACAGTCCCACGTGA
T</p>
</div>
</details><p><em><strong>Q: How much information contained in the header of the FASTA file can you decipher?</strong></em></p>
<p><em><strong>Q: What gene specifically was sequenced for these two species?</strong></em></p>
<blockquote class="epigraph">
<div><p>With few exceptions, the cells of all eukaryotic species contain mitochondria. The mitochondrial genome comprises a double-stranded DNA molecule that accounts for 1% to 2% of the total DNA in mammalian cells. The mitochondrial genome encodes 13 essential oxidative phosphorylation subunit proteins/polypeptides. It also encodes two rRNAs (12S rRNA and 16S rRNA) and 22 tRNAs that are required for mitochondrial protein synthesis <a class="reference external" href="https://www.nature.com/articles/srep04089#Bib1">13,14,15,16</a>. Mitochondria possess their own organelle-specific DNA replication, transcription and translation systems <a class="reference external" href="https://www.nature.com/articles/srep04089#Bib1">13,14,15,16</a>.</p>
<p>Typical animal mtDNA has a high mutation rate and an exceptional organizational economy, with rare non-coding segments. The accelerated evolutionary rate of animal mtDNA implies that significant amounts of sequence variation could be observed in closely related species—a useful feature for species identification procedures.</p>
<p class="attribution">—Yang et al (2014)<a class="footnote-reference brackets" href="#yang2014" id="id1">1</a></p>
</div></blockquote>
<p><em><strong>Q: Given the sequences shown from <a class="reference external" href="#fasta-files">the two FASTA files</a>, why would it be difficult to determine differences in the genetic sequence between species?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
</div>
<div class="section" id="sequence-alignment">
<h2>Sequence Alignment<a class="headerlink" href="#sequence-alignment" title="Permalink to this headline">¶</a></h2>
<p>The online webtool <a class="reference external" href="https://www.ebi.ac.uk/Tools/msa/muscle/">MUSCLE</a> was developed to align genomic sequences across species. The output of the alignment procedure looks like the following:</p>
<p><a id='alignment'></a></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>Hawksbill        AACTAT-TTAACTTACC-ACTCTAAAGACTTATAATTTACTAGTTC-TGATCCATATTTT
Loggerhead       AACTAT-TATATTTACC-ACCCTAAAGACTTATAATTAACTAGTTC-TGATCCATATTTT
Olive            AACTAT-CACACTTACT-ACCCTAAAGACTTATAACTTACTAGTTC-TGATCCATATTTT
Kemp’s           AACTAT-TACACTTACT-ACCCTAAAGACTTATAACTTACTAGTTC-TGATCCATATTTT
Leatherback      AACTACACATCCACACCTAATCTAAGGACTTATAACCAACTAG-ACTTGATCCATATTTT
Flatback         AACTAT-CATACTCACCCACCCCAAGGACCTATAACTAACTAATACTTGACCTATATTTT
Green            AACTAT-CATACCCACTCACTCTAAGGACCTATAACTAATTAGTACTTGACCTATATTTT
                 *****         **  *  * ** *** *****   * **   * *** * *******
</pre></div>
</div>
<p><em><strong>Q: What do the stars represent?</strong></em></p>
<p><em><strong>Q: Compute a metric of “relatedness” between Hawksbill and Loggerhead turtles based on sequence (dis)similarity.</strong></em></p>
<p><em><strong>Q: Use the same method ot compute the “relatedness” between Hawksbill and Green turtles</strong></em></p>
</div>
<div class="section" id="from-relatedness-to-trees">
<h2>From Relatedness to Trees<a class="headerlink" href="#from-relatedness-to-trees" title="Permalink to this headline">¶</a></h2>
<p>In a phylogenetic tree, the tips of branches represent <em>extant</em> (versus <em>extinct</em>) species. The tree is <em>rooted</em> with the <em>last common ancestor</em> (LCA) of all extant species on the tree. Each branch point from the root is a point of evolutionary <em>divergence</em>.</p>
<div class="figure align-default" id="sample-tree">
<a class="bg-primary mb-1 reference internal image-reference" href="../_images/Sample-Tree.png"><img alt="fishy" class="bg-primary mb-1" src="../_images/Sample-Tree.png" style="width: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 6 </span><span class="caption-text">An example phylogenetic tree.</span><a class="headerlink" href="#sample-tree" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Q: How many extant species are there in the tree (and what are their labels)?</strong></em></p>
<p><em><strong>Q: How many points of divergence are there?</strong></em></p>
<p><em><strong>Q: If you used this tree as a phylogeny for the three turtle species for which you calculated relatedness (Hawksbill, Loggerhead, and Green), which letter (A, B, or C) would correspond to each turtle species?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 5 min</p>
</div></blockquote>
<p><em><strong>Q: Based on all of the information provided so far, how would you define ‘phylogeny’?</strong></em></p>
<p><em><strong>Q: Why is mitochondrial DNA particularly useful for assembling a phylogeny?</strong></em></p>
<p><em><strong>Q: Why is a phylogeny a hypothesis? What information could you gather that might change the tree that you constructed?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
<p>Based on the <a class="reference external" href="#alignment">MUSCLE alignment of the 7 turtle species</a>, we can use a computer algorithm to calculate relatedness and generate a drawing of the most likely phylogeny.</p>
<div class="figure align-default" id="turtles-cladogram-black">
<a class="bg-primary mb-1 reference internal image-reference" href="../_images/turtles_cladogram_black.png"><img alt="fishy" class="bg-primary mb-1" src="../_images/turtles_cladogram_black.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 7 </span><span class="caption-text">A phylogenetic tree of 7 turtle species. Relatedness and branch lengths were calculated from the MUSCLE alignement data of the 16S ribosomal RNA partial gene sequence using <a class="reference external" href="https://biopython.org/">BioPython</a>.</span><a class="headerlink" href="#turtles-cladogram-black" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="from-trees-to-behavioral-evolutionary-history">
<h2>From Trees to Behavioral Evolutionary History<a class="headerlink" href="#from-trees-to-behavioral-evolutionary-history" title="Permalink to this headline">¶</a></h2>
<p>If you search the <a class="reference external" href="https://animaldiversity.org/">Animal Diversity Web</a> (for example), you will find that, of the 7 turtle species you have just examined, all but one are classified as <em>solitary</em>. The green turtle (<em>Chelonia mydas</em>) is classified as “colonial”<a class="footnote-reference brackets" href="#colonial" id="id2">2</a>.</p>
<div class="figure align-default" id="turtles-cladogram-character-map">
<a class="bg-primary mb-1 reference internal image-reference" href="../_images/turtles_cladogram_character-map.png"><img alt="fishy" class="bg-primary mb-1" src="../_images/turtles_cladogram_character-map.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 8 </span><span class="caption-text">A character mapping of social behavior on the phylogeny of 7 turtle species. Green = colonial. Orange = solitary.</span><a class="headerlink" href="#turtles-cladogram-character-map" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Q: Based on the information given, define ‘character mapping’.</strong></em></p>
<p>From a phylogenetic perspective, we can understand why the green turtle colonial. There are two alternative explanations: that colonialism is a phylogenetically <em>inherited</em> behavior or that colonialism is a phylogenetically <em>derived</em> behavior.</p>
<p>Examine the following two <em>ancestral state reconstruction</em> hypotheses and their graphical representation.</p>
<p><strong>Hypothesis #1</strong>: The LCA of this turtle phylogeny was colonial. A consequence of this hypothesis would be that colonialism is an evolutionarily inherited behavior in green turtles.</p>
<div class="figure align-default" id="turtles-cladogram-lca1">
<a class="bg-primary mb-1 reference internal image-reference" href="../_images/turtles_cladogram_LCA1.png"><img alt="fishy" class="bg-primary mb-1" src="../_images/turtles_cladogram_LCA1.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 9 </span><span class="caption-text">A phylogenetic tree of 7 turtle species. Relatedness and branch lengths were calculated from the MUSCLE alignement data of the 16S ribosomal RNA partial gene sequence using BioPython.</span><a class="headerlink" href="#turtles-cladogram-lca1" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Hypothesis #2</strong></em>: The LCA of this turtle phylogeny was solitary. A consequence of this hypothesis would be that colonialism is an evolutionarily derived behavior in green turtles.</p>
<div class="figure align-default" id="turtles-cladogram-lca0">
<a class="bg-primary mb-1 reference internal image-reference" href="../_images/turtles_cladogram_LCA0.png"><img alt="fishy" class="bg-primary mb-1" src="../_images/turtles_cladogram_LCA0.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 10 </span><span class="caption-text">A phylogenetic tree of 7 turtle species. Relatedness and branch lengths were calculated from the MUSCLE alignement data of the 16S ribosomal RNA partial gene sequence using BioPython.</span><a class="headerlink" href="#turtles-cladogram-lca0" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Q: Under which ancestral state reconstruction hypothesis (1 or 2) would colonialism be an inherited trait in green turtles?</strong></em></p>
<p><em><strong>Q: What is the difference between an ‘ancestral state reconstruction’ and a ‘character mapping’?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
<p>We can provide evidence for/against each of these alternative hypotheses about the LCA behavioral trait by using an <em>argument of parsimony</em>.</p>
<p><em><strong>Q: Which requires more events/steps? Evolutionary change or evolutionary stasis?</strong></em></p>
<p><em><strong>Q: If the LCA of the turtles in Figure ______ was solitary (hypothesis #2), what is the least number of evolutionary changes that would need to occur to result in the correct behavioral traits of all 7 extant species?</strong></em></p>
<p><em><strong>Q: If the LCA of the turtles in Figure ______ was colonial (hypothesis #1), what is the least number of evolutionary changes that would need to occur to result in the correct behavioral traits of all 7 extant species?</strong></em></p>
<p><em><strong>Q: Which alternate hypothesis would require more evolutionary changes?</strong></em></p>
<p><em><strong>Q: All else being equal, which hypothesis results in a less likely series of events: “LCA was solitary” or “LCA was colonial”?</strong></em></p>
<p><em><strong>Q: If the method of parsimony is used to reject the less likely hypothesis, then what would be the result of your analysis? In other words, how would you interpret your results in terms of determining the behavioral trait exhibited by the LCA of these 7 turtles?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
<p><em><strong>Q: What is the minimum subset of this phylogenetic tree that you would need to provide evidence that colonialism is a derived trait in green turtles?</strong></em></p>
<p><em><strong>Q: If there are species of turtles missing from this phylogeny, how could that change the results of your ancestral state reconstruction analysis?</strong></em></p>
<p><em><strong>Q: If no one had yet characterized the behavior of some extant species within your phylogeny, how would that add uncertainty to your analysis and interpretation of the results?</strong></em></p>
<p>Many published phylogenetic trees exist that you can explore without needing to analyze any genetics. For example, <a class="reference external" href="https://tree.opentreeoflife.org/opentree/opentree13.4&#64;ott66469/Cheloniidae">here is the entry on OpenTree that includes all 7 turtle species you have just examined</a>. This tree is rooted at <em>Cheloniidae</em> (the sea turtles), which is a <em>monophyletic group</em>. In this web interface, branches of the tree are sometimes collapsed for visual clarity. Clicking on each branch line in the tree reveals how many extant species derive from that branch. Clicking on each node in the tree roots the visualization at that node (often expanding/collapsing sets of branches as the view changes). Practice navigating around the tree.</p>
<p><em><strong>Q: How many other species of turtles should have been included in Figure 2 to generate a complete phylogeny of sea turtles?</strong></em></p>
<p>⏹️ STOP here for today</p>
</div>
<div class="section" id="additional-resources">
<h2>Additional Resources<a class="headerlink" href="#additional-resources" title="Permalink to this headline">¶</a></h2>
<p>Data for the turtle phylogeny was collected by Lindsay Taylor as part of their <a class="reference external" href="https://taylor-lindsay.github.io/phylogenetics/">example jupyter notebooks implementation of BioPython</a></p>
<hr class="footnotes docutils" />
<dl class="footnote brackets">
<dt class="label" id="yang2014"><span class="brackets"><a class="fn-backref" href="#id1">1</a></span></dt>
<dd><p>Yang, L., Tan, Z., Wang, D. et al. <a class="reference external" href="https://doi.org/10.1038/srep04089">Species identification through mitochondrial rRNA genetic analysis</a>. Sci Rep 4, 4089 (2014).</p>
</dd>
<dt class="label" id="colonial"><span class="brackets"><a class="fn-backref" href="#id2">2</a></span></dt>
<dd><p>“Used loosely to describe any group of organisms living together or in close proximity to each other - for example nesting shorebirds that live in large colonies. More specifically refers to a group of organisms in which members act as specialized subunits (a continuous, modular society) - as in clonal organisms.” - <a class="reference external" href="https://animaldiversity.org/accounts/Chelonia_mydas/#behavior">ADW</a>”</p>
</dd>
</dl>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./levels-of-analysis"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            
                <!-- Previous / next buttons -->
<div class='prev-next-area'> 
    <a class='left-prev' id="prev-link" href="../observation-and-analysis/analysis.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Analysis</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="evolution.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Evolution</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            
        </div>
    </div>
    <footer class="footer">
  <p>
    
      By Dr. Krista Perks<br/>
    
        &copy; Copyright 2021.<br/>
  </p>
</footer>
</main>


      </div>
    </div>
  
  <script src="../_static/js/index.be7d3bbb2ef33a8344ce.js"></script>

  </body>
</html>