
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Phylogeny &#8212; Comparative Animal Behavior</title>
    
  <link href="../_static/css/theme.css" rel="stylesheet">
  <link href="../_static/css/index.ff1ffe594081f20da1ef19478df9384b.css" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      

    
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-book-theme.css?digest=c3fdc42140077d1ad13ad2f1588a4309" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../_static/js/index.be7d3bbb2ef33a8344ce.js">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/sphinx-book-theme.d59cb220de22ca1c485ebbdc042f0030.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Evolution" href="evolution.html" />
    <link rel="prev" title="Ultimate" href="ultimate.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../_static/Moffett-ants.jpg" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">Comparative Animal Behavior</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../welcome.html">
   Welcome
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  General
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../syllabus.html">
   Syllabus
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../schedule.html">
   Schedule
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../assessment-sum/summative-assessment.html">
   Summative Assessment
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment-example.html">
     Example
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_components-observation-analysis.html">
     Components of Behavior, Observation, and Analysis
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_phylogeny-evolution.html">
     Phylogeny and Evolution
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_neural-hormonal-genetic-development.html">
     Neural, Hormonal, Genetic, and Development
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../assessment-sum/assessment_understanding-behavior.html">
     Understanding Behavior
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../assessment-sum/capstone.html">
   Capstone
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../genindex.html">
   Terms, Topics, Concepts
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../ethical-considerations.html">
   Ethical Considerations for Animal Use
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Describing Behavior
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/components.html">
   Components
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/observation.html">
   Observation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../observation-and-analysis/analysis.html">
   Analysis
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Levels of Analysis
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1 current active has-children">
  <a class="reference internal" href="ultimate.html">
   Ultimate
  </a>
  <input checked="" class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="current">
   <li class="toctree-l2 current active">
    <a class="current reference internal" href="#">
     Phylogeny
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="evolution.html">
     Evolution
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="proximate.html">
   Proximate
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="neural.html">
     Neural Mechanisms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="hormonal.html">
     Hormonal Mechanisms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="genetic.html">
     Genetic Mechanisms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="endogenous-rhythms.html">
     Endogenous Rhythms
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="development.html">
     Development
    </a>
   </li>
  </ul>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Understanding Behavior
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/navigation.html">
   Navigation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../understanding-behavior/escape.html">
   Escape
  </a>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../understanding-behavior/communication.html">
   Communication
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-4" name="toctree-checkbox-4" type="checkbox"/>
  <label for="toctree-checkbox-4">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/communication_proximate.html">
     Communication: Proximate
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/communication_ultimate.html">
     Communication: Ultimate
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../understanding-behavior/social-spectrum.html">
   Social Spectrum
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-5" name="toctree-checkbox-5" type="checkbox"/>
  <label for="toctree-checkbox-5">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/social-spectrum_cooperation.html">
     Grouping and Cooperation
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/social-spectrum_eusociality.html">
     Eusociality
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../understanding-behavior/family-life.html">
   Family Life
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-6" name="toctree-checkbox-6" type="checkbox"/>
  <label for="toctree-checkbox-6">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/family-life_sexual-selection.html">
     Sexual Selection
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/family-life_mating-systems.html">
     Mating Systems
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../understanding-behavior/family-life_parental-care.html">
     Parental Care
    </a>
   </li>
  </ul>
 </li>
</ul>

    </div>
</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../_sources/levels-of-analysis/phylogeny.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
                onclick="printPdf(this)" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/neurologic/Comparative-Animal-Behavior/issues"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        
        
    </div>
</div>

            <!-- Full screen (wrap in <a> to have style consistency -->

<a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
        data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
        title="Fullscreen mode"><i
            class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show noprint">
            
            <div class="tocsection onthispage pt-5 pb-3">
                <i class="fas fa-list"></i> Contents
            </div>
            <nav id="bd-toc-nav" aria-label="Page">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genetic-sequences">
   Genetic Sequences
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-relatedness-to-trees">
   From Relatedness to Trees
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sequence-alignment">
   Sequence Alignment
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-trees-to-behavioral-evolutionary-history">
   From Trees to Behavioral Evolutionary History
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#case-study-vocal-learning">
     Case Study: Vocal Learning
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#additional-resources">
   Additional Resources
  </a>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Phylogeny</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genetic-sequences">
   Genetic Sequences
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-relatedness-to-trees">
   From Relatedness to Trees
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sequence-alignment">
   Sequence Alignment
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-trees-to-behavioral-evolutionary-history">
   From Trees to Behavioral Evolutionary History
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#case-study-vocal-learning">
     Case Study: Vocal Learning
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#additional-resources">
   Additional Resources
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            
              <div>
                
  <div class="tex2jax_ignore mathjax_ignore section" id="phylogeny">
<h1>Phylogeny<a class="headerlink" href="#phylogeny" title="Permalink to this headline">¶</a></h1>
<!-- 
<hr>

> {sub-ref}`wordcount-words` words | {sub-ref}`wordcount-minutes` min read

<hr> -->
<p><a class="reference internal" href="#phylogeny-day1"><span class="std std-ref">Day 1</span></a>; <a class="reference external" href="https://docs.google.com/document/d/1Zny7rFlzbYkPHoAMDLy6aPTXH-bod1ODxdhiLQz9AIg/edit?usp=sharing">Responses template for use on computer if desired</a><br />
<a class="reference internal" href="#phylogeny-day2"><span class="std std-ref">Day 2</span></a>; <a class="reference external" href="https://docs.google.com/document/d/1mqKfBa3jPaanVQRGTZ3KtieONeU9ocnquhBuTntxnUM/edit?usp=sharing">Responses template for use on computer if desired</a></p>
<div class="admonition-why admonition" id="index-0">
<p class="admonition-title">Why?</p>
<p>Animal behaviorists use <strong>phylogenies</strong> to investigate the development of behavior across generations (evolutionary timescales).
Critically, a <strong>phylogeny</strong> is a hypothesis. Phylogenies can be tested and falsified, and can change given new evidence or analytic techniques.
Phylogenies can be based on any animal trait. Commonly, in studying animal behavior, you will encounter phylogenies based on <em>genetic sequence data</em>.</p>
</div>
<p>Today and tomorrow, you will be working on understanding what phylogenies are and phylogenetic analyses that are used to study animal behavior. All of the information that you need for today is contained on this page. However, this page also links you to tools that you can use in the future to study the phylogenetic basis of behavior.</p>
<div class="section" id="genetic-sequences">
<span id="phylogeny-day1"></span><h2>Genetic Sequences<a class="headerlink" href="#genetic-sequences" title="Permalink to this headline">¶</a></h2>
<p>All animals have genes (generally, sequences of <em>nucleutides</em>, usually abbreviated <em>A</em>, <em>T</em>, <em>G</em>, and <em>C</em>). <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/23193287">GenBank</a> is the NIH genetic sequence database, an annotated collection of all publicly available DNA sequences. The <a class="reference external" href="https://www.ncbi.nlm.nih.gov/nucleotide/">nucleotide search</a> lets you search for species names and genes. Here, I have provided you with example sequences from the GenBank to refer to.<a class="footnote-reference brackets" href="#genbank" id="id1">1</a></p>
<!-- ---
> ⏳ 5 min 
 -->
<p><a id='fasta-files'></a>
Below are two <em>FASTA</em> files downloaded from GenBank. “FASTA” is a common file format used to store genetic sequence data. The files are very simple; they contain a <em>header</em> followed by a sequence of nucleotide codes. The file header provides <em>metadata</em> about the file (like which gene was sequenced and the sequence length). Click on the file headers to see its contents.</p>
<details class="sphinx-bs dropdown card mb-3">
<summary class="summary-title card-header">
HQ377551.1 <em>Chelonia mydas</em> isolate Satang_4 16S ribosomal RNA gene, partial sequence; mitochondrial; length 510<div class="summary-down docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-down" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M5.22 8.72a.75.75 0 000 1.06l6.25 6.25a.75.75 0 001.06 0l6.25-6.25a.75.75 0 00-1.06-1.06L12 14.44 6.28 8.72a.75.75 0 00-1.06 0z"></path></svg></div>
<div class="summary-up docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-up" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M18.78 15.28a.75.75 0 000-1.06l-6.25-6.25a.75.75 0 00-1.06 0l-6.25 6.25a.75.75 0 101.06 1.06L12 9.56l5.72 5.72a.75.75 0 001.06 0z"></path></svg></div>
</summary><div class="summary-content card-body docutils">
<p class="card-text">GGCCGCGGTATCCTAACCGTGCAAAGGTAGCGTAATCACTTGTCTTTTAAATAAAGACTAGAATGAATGG<br />
CCAAACGAGGTTCTACCTGTCTCTTACAAACAATCAGTGAAATTGATCTCCCCGTGCAAAAGCGGGGATA
ACACTATAAGACGAGAAGACCCTGTGGAACTTTAAATACAGATCAACTATCATACCCACTCACTCTAAGG<br />
ACCTATAACTAATTAGTACTTGACCTATATTTTTGGTTGGGGCGACCTCGGAGTAAAACAAAACCTCCGA<br />
AAAAAGAATACACTTCTTAACCTAGACCCACAATTCAAAGTGCCAACGGCAAAATGATCCAATATATTTG<br />
ATCAACGAACCAAGCTACCCCAGGGATAACAGCGCAATCCCATCCTAGAGTTCCTATCGACGATGGGGTT<br />
TACGACCTCGATGTTGGATCAGGACATCCTGATGGTGCAACCGCTATCAAGGGTTCGTTTGTTCAACGAT<br />
TAACAGTCCTACGTGATCTG</p>
</div>
</details><details class="sphinx-bs dropdown card mb-3">
<summary class="summary-title card-header">
FJ039971.1 <em>Eretmochelys imbricata</em> isolate EI_Atlantic 16S ribosomal RNA gene, partial sequence; mitochondrial; length 560<div class="summary-down docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-down" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M5.22 8.72a.75.75 0 000 1.06l6.25 6.25a.75.75 0 001.06 0l6.25-6.25a.75.75 0 00-1.06-1.06L12 14.44 6.28 8.72a.75.75 0 00-1.06 0z"></path></svg></div>
<div class="summary-up docutils">
<svg version="1.1" width="24" height="24" class="octicon octicon-chevron-up" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M18.78 15.28a.75.75 0 000-1.06l-6.25-6.25a.75.75 0 00-1.06 0l-6.25 6.25a.75.75 0 101.06 1.06L12 9.56l5.72 5.72a.75.75 0 001.06 0z"></path></svg></div>
</summary><div class="summary-content card-body docutils">
<p class="card-text">GCCTCTAGCAACAACAAGTATTAGAGGTAATGCCTGCCCAGTGACACTGTTAAACGGCCGCGGTATCCTA
ACCGTGCAAAGGTAGCGTAATCACTTGTCTTTTAAATAAAGACTAGAATGAATGGCCAAACGAGGTTCCA
CCTGTCTCTTACAAACAATCAGTGAAATTGGTCTCCCCGTGCAAAAGCGAGGATAGCACTATAAGACGAG
AAGACCCTGTGGAACTTTAAATATAAATCAACTATTTAACTTACCACTCTAAAGACTTATAATTTACTAG
TTCTGATCCATATTTTTGGTTGGGGTGACCTCGGAGAAAAACAAAACCTCCGAAAAAAGAACATATCTTC
TTAACCTAGACCCACAACTCAAAGTGCCAACGGAAAAATGATCCAATATATTTGATCAACGAACCAAGCT
ACCCCAGGGATAACAGCGCAATCCCATCTTAGAGTCCATATCGACGATGGGGTTTACGACCTCGATGTTG
GATCAGGACATCCTGATGGTGCAACCGCTATCAAGGGTTCGTTTGTTCAACGATTAACAGTCCCACGTGAT</p>
</div>
</details><!-- 
***Q1: Given the information in the header file, what gene was sequenced?*** 

***Q2: How long is the sequenced gene for each species?*** 


⏸️ PAUSE here for class-wide discussion

---
 -->
<p>From <a class="reference external" href="https://doi.org/10.1038/srep04089">Yang et al (2014)</a>: “With few exceptions, the cells of all eukaryotic species contain mitochondria. The mitochondrial genome comprises a double-stranded DNA molecule that accounts for 1% to 2% of the total DNA in mammalian cells. Typical animal mitochondrial DNA (<em><strong>mtDNA</strong></em>) has a high mutation rate. The accelerated evolutionary rate of animal mtDNA implies that significant amounts of sequence variation could be observed in closely related species—a useful feature for species identification procedures.”<a class="footnote-reference brackets" href="#yang-2014" id="id2">2</a></p>
</div>
<div class="section" id="from-relatedness-to-trees">
<h2>From Relatedness to Trees<a class="headerlink" href="#from-relatedness-to-trees" title="Permalink to this headline">¶</a></h2>
<p id="index-1">In a tree representation of a <strong>phylogeny</strong> (Figure 9), the tips of branches represent <em>extant</em> (versus <em>extinct</em>) species. The tree is <em>rooted</em> with the <em><strong>Last Common Ancestor</strong></em> (<strong>LCA</strong>) of all extant species on the tree (the evolutionarily oldest species of the tree). Each branch point from the root is a point of evolutionary <em>divergence</em>. We talk about species being closer to each other on a phylogenetic tree as being more <em>related</em> (having a higher <em>relatedness</em>).</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
<div class="figure align-default" id="id11">
<a class="reference internal image-reference" href="../_images/Sample-Tree.png"><img alt="../_images/Sample-Tree.png" src="../_images/Sample-Tree.png" style="width: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 9 </span><span class="caption-text">An example phylogenetic tree.</span><a class="headerlink" href="#id11" title="Permalink to this image">¶</a></p>
</div>
<p>The following table reports the number of mtDNA sequence differences between extant species in the example phylogeny (Figure 9).</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="text-align:center head"><p></p></th>
<th class="text-align:center head"><p>B</p></th>
<th class="text-align:center head"><p>C</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td class="text-align:center"><p><strong>A</strong></p></td>
<td class="text-align:center"><p>5</p></td>
<td class="text-align:center"><p>23</p></td>
</tr>
<tr class="row-odd"><td class="text-align:center"><p><strong>B</strong></p></td>
<td class="text-align:center"><p>-</p></td>
<td class="text-align:center"><p>23</p></td>
</tr>
</tbody>
</table>
<p><em><strong>Q1: How many extant species are there in the tree?</strong></em></p>
<p><em><strong>Q2: How many points of divergence are there?</strong></em></p>
<p><em><strong>Q3: If there are fewer branch points between two species, do they have more or fewer differences in their mtDNA sequence?</strong></em></p>
<p><em><strong>Q4: Given the sequences shown in <a class="reference external" href="#fasta-files">the two FASTA files</a> that you already examined, why would it be difficult to determine differences in the genetic sequence between species?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
</div>
<div class="section" id="sequence-alignment">
<h2>Sequence Alignment<a class="headerlink" href="#sequence-alignment" title="Permalink to this headline">¶</a></h2>
<p>The online webtool <a class="reference external" href="https://www.ebi.ac.uk/Tools/msa/muscle/">MUSCLE</a> was developed to align genomic sequences across species. The following is the output of the MUSCLE alignment procedure for a genetic sequence (16S ribosomal RNA partial gene sequence) from 7 species of turtles:</p>
<p><a id='alignment'></a></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>Hawksbill        AACTAT-TTAACTTACC-ACTCTAAAGACTTATAATTTACTAGTTC-TGATCCATATTTT
Loggerhead       AACTAT-TATATTTACC-ACCCTAAAGACTTATAATTAACTAGTTC-TGATCCATATTTT
Olive            AACTAT-CACACTTACT-ACCCTAAAGACTTATAACTTACTAGTTC-TGATCCATATTTT
Kemp’s           AACTAT-TACACTTACT-ACCCTAAAGACTTATAACTTACTAGTTC-TGATCCATATTTT
Leatherback      AACTACACATCCACACCTAATCTAAGGACTTATAACCAACTAG-ACTTGATCCATATTTT
Flatback         AACTAT-CATACTCACCCACCCCAAGGACCTATAACTAACTAATACTTGACCTATATTTT
Green            AACTAT-CATACCCACTCACTCTAAGGACCTATAACTAATTAGTACTTGACCTATATTTT
                 *****         **  *  * ** *** *****   * **   * *** * *******
</pre></div>
</div>
<p><em><strong>Q5: What does the star mean for a column of letters (molecules in the gene sequence)?</strong></em></p>
<p><em><strong>Q6: Define and compute a metric of relatedness (genetic similarity) between Hawksbill and Loggerhead turtles based on sequence (dis)similarity.</strong></em></p>
<p><em><strong>Q7: Use the same method to compute the relatedness between Hawksbill and Green turtles.</strong></em></p>
<div class="margin sidebar">
<p class="sidebar-title">Figure 9</p>
<img alt="../_images/Sample-Tree.png" src="../_images/Sample-Tree.png" />
</div>
<p><em><strong>Q8: If you used the example phylogeny (Figure 9) for these three turtle species (Hawksbill, Loggerhead, and Green), which letter (A, B, or C) would most likely correspond to each turtle species?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 5 min</p>
</div></blockquote>
<p><em><strong>Q9: Based on all of the information provided so far, how would you define ‘phylogeny’?</strong></em></p>
<p><em><strong>Q10: Remember that a phylogeny is a hypothesis. What is an ‘alternative hypothesis’ for the example phylogeny (Figure 9)?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<p>Readily available computer algorithms can calculate relatedness from <em>aligned</em> genetic sequence data and generate a drawing of the most likely phylogeny.</p>
<div class="figure align-default" id="id12">
<a class="reference internal image-reference" href="../_images/turtles_cladogram_black.png"><img alt="../_images/turtles_cladogram_black.png" src="../_images/turtles_cladogram_black.png" style="width: 700px;" /></a>
<p class="caption"><span class="caption-number">Fig. 10 </span><span class="caption-text">A phylogenetic tree of 7 turtle species. Relatedness and branch lengths were calculated from the MUSCLE alignement data of the 16S ribosomal RNA partial gene sequence using <a class="reference external" href="https://biopython.org/">BioPython</a>.</span><a class="headerlink" href="#id12" title="Permalink to this image">¶</a></p>
</div>
<p>Many published phylogenetic trees exist that you can explore without needing to obtain or analyze any genetics yourself. For example, <a class="reference external" href="https://tree.opentreeoflife.org/opentree/opentree13.4&#64;ott66469/Cheloniidae">here is the entry on OpenTree that includes all 7 turtle species you have just examined</a>. This tree is rooted at <em>Cheloniidae</em> (the sea turtles), which is a <em>monophyletic group</em>.</p>
<blockquote>
<div><p>In this web interface, branches of the tree are sometimes collapsed for visual clarity. Clicking on a branch line in the tree (or the “<em>i</em>” next to a node) reveals how many extant species derive from that branch. Clicking on each node in the tree roots the visualization at that node (often expanding/collapsing sets of branches as the view changes).<br />
Clicking on the <code class="docutils literal notranslate"><span class="pre">i</span></code> icon next to a node brings up a menu with a <code class="docutils literal notranslate"><span class="pre">Search</span> <span class="pre">EOL</span> <span class="pre">for</span> <span class="pre">'node</span> <span class="pre">name'</span></code> link. <em><strong>EOL</strong></em> is the “Encyclopedia of Life”. This non-profit is a great resource for biodiversity information.</p>
</div></blockquote>
<p>Practice navigating around the tree.</p>
<p>Think about an animal whose behavior you are interested in and practice researching its phylogenic history and relationships.</p>
<p>⏹️ STOP here for today</p>
<hr class="docutils" />
</div>
<div class="section" id="from-trees-to-behavioral-evolutionary-history">
<span id="phylogeny-day2"></span><h2>From Trees to Behavioral Evolutionary History<a class="headerlink" href="#from-trees-to-behavioral-evolutionary-history" title="Permalink to this headline">¶</a></h2>
<p>If you search the <a class="reference external" href="https://animaldiversity.org/">Animal Diversity Web</a> (for example), you will find that, of the 7 turtle species you have just examined, all but one are classified as <em>solitary</em>. The green turtle (<em>Chelonia mydas</em>) is classified as <em>colonial</em><a class="footnote-reference brackets" href="#colonial" id="id3">3</a>.</p>
<p>From a phylogenetic perspective, why is the green turtle colonial? Either colonialism is an <em>inherited</em> behavior or colonialism is a <em>derived</em> behavior.</p>
<p>In order to determine whether colonialism is derived or inherited, we need an <strong>ancestral state reconstruction</strong>. To create this reconstruction, we need to:</p>
<ol class="simple">
<li><p>make a <em><strong>character mapping</strong></em></p></li>
<li><p>test all <em><strong>alternative hypotheses for the ancestral state reconstruction</strong></em>.</p></li>
</ol>
<p>So what is a character mapping, an ancestral state reconstruction, the hypothesis testing, etc?   We will work through the next two sections to learn about the process…</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 10 min</p>
</div></blockquote>
<div class="figure align-default" id="id13">
<a class="reference internal image-reference" href="../_images/turtles-charmap-lca.png"><img alt="../_images/turtles-charmap-lca.png" src="../_images/turtles-charmap-lca.png" style="width: 800px;" /></a>
<p class="caption"><span class="caption-number">Fig. 11 </span><span class="caption-text">Graphical representations of character mapping and ancestral state reconstruction. To construct the turtle phylogeny, relatedness and branch lengths were calculated from the <em>MUSCLE</em> alignement data of the 16S ribosomal RNA partial gene sequence using <em>BioPython</em>.<br />
<strong>A)</strong> <em>Character mapping</em> of social behavior on the turtle phylogeny.<br />
Green = colonial. Orange = solitary.<br />
<strong>B)</strong> <em>Ancestral state reconstruction</em> under <em><strong>Hypothesis #1</strong></em> on the turtle phylogeny.<br />
<strong>C)</strong> <em>Ancestral state reconstruction</em> under <em><strong>Hypothesis #2</strong></em> on the turtle phylogeny.</span><a class="headerlink" href="#id13" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Q1: Based on the information given, what do you think ‘character mapping’ is?</strong></em></p>
<p><em><strong>Q2: Under Hypothesis #1 for the ancestral state reconstruction, the last common ancestor (LCA) of this turtle phylogeny exhibited ______________ behavior (fill in the blank).</strong></em></p>
<p><em><strong>Q3: Under Hypothesis #2 for the ancestral state reconstruction, the LCA of this turtle phylogeny exhibited ______________ behavior (fill in the blank).</strong></em></p>
<p><em><strong>Q4: How is character mapping different than ancestral state reconstruction?</strong></em></p>
<p><em><strong>Q5: Under which ancestral state reconstruction hypothesis (1 or 2) would colonialism be an inherited (versus derived) trait in green turtles?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 15 min</p>
</div></blockquote>
<p>One common way to assess alternative hypotheses for the LCA behavioral trait is to use an <em>argument of</em> <em><strong>parsimony</strong></em>. Answer the following questions to understand the logic of making an argument about LCA behavior based on <em><strong>parsimony</strong></em>.</p>
<div class="margin sidebar">
<p class="sidebar-title">copy of Figure 11 for quick reference</p>
<img alt="../_images/turtles-charmap-lca_B-C.png" src="../_images/turtles-charmap-lca_B-C.png" />
</div>
<p><em><strong>Q6: If the LCA of the turtles in Figure 11 was solitary, what is the least number of total evolutionary changes that would need to occur (across the tree) to result in the correct behavioral traits of all 7 extant species?</strong></em></p>
<p><em><strong>Q7: If the LCA of the turtles in Figure 11 was colonial, what is the least number of total evolutionary changes that would need to occur to result in the correct behavioral traits of all 7 extant species?</strong></em></p>
<p><em><strong>Q8: Which alternative hypothesis would require more evolutionary changes?</strong></em></p>
<p>By an argument of parsimony, we would conclude that it is less likely that the LCA was colonial. Therefore, the evidence supports Hypothesis #1 less than Hypothesis #2.</p>
<p><em><strong>Q9: From that conclusion, which would you infer is more likely: evolutionary change or evolutionary stasis?</strong></em></p>
<p><em><strong>Q10: Create your definition for the “argument of parsimony”.</strong></em></p>
<p><em><strong>Q11: If no one had yet characterized the behavior of Flatback and Leatherback turtles, would it change the results of your analysis? How would you handle this unknown data in your analysis?</strong></em></p>
<p>⏸️ PAUSE here for class-wide discussion</p>
<hr class="docutils" />
<p>Importantly, by reconstructing the evolutionary history of behaviors, we can determine whether extant species that share the behavior <em>converged</em> on the same adaptation or whether they <em>inherited</em> from a common ancestor. Similarly, points of <em>divergence</em> can be significant to our understanding of the behavior.</p>
<div class="section" id="case-study-vocal-learning">
<h3>Case Study: Vocal Learning<a class="headerlink" href="#case-study-vocal-learning" title="Permalink to this headline">¶</a></h3>
<div class="figure align-default" id="id14">
<a class="reference internal image-reference" href="../_images/phylogeny-vocal-learning-comparison.png"><img alt="../_images/phylogeny-vocal-learning-comparison.png" src="../_images/phylogeny-vocal-learning-comparison.png" style="width: 900px;" /></a>
<p class="caption"><span class="caption-number">Fig. 12 </span><span class="caption-text">Phylogeny of <em>Neoaves</em> (By clicking on the image you can zoom in).<br />
<strong>A)</strong> From the <em>Animal Behavior, 9th edition</em> textbook by Rubenstein and Alcock (2009). Phylogeny data generated using DNA-DNA hybridization<a class="footnote-reference brackets" href="#dna-hybridization" id="id4">4</a> (basically an analog version of the MUSCLE alignment algorithm and published by Eliot A. Brenowitz in <em>Evolution of the vocal control system in the avian brain</em><a class="footnote-reference brackets" href="#brenowitz-1991" id="id5">5</a>. Character mapping of vocal learning is highlighted in green, blue, and red.<br />
<strong>B)</strong> From Suh et al (2011)<a class="footnote-reference brackets" href="#suh-2011" id="id6">6</a>. Phylogeny data generated using Mesozoic <em>retroposons</em><a class="footnote-reference brackets" href="#retroposons" id="id7">7</a> and analysis utilizing <em>maximum parsimony</em>. New taxa <em>Eufalconimorphae</em> (falcons+parrots+passerines) and <em>Psittacopasserae</em> (parrots+passerines) are highlighted.</span><a class="headerlink" href="#id14" title="Permalink to this image">¶</a></p>
</div>
<hr class="docutils" />
<blockquote>
<div><p>⏳ 15 min</p>
</div></blockquote>
<div class="figure align-default" id="id15">
<a class="reference internal image-reference" href="../_images/phylogeny-vocal-learning-comparison-drawn.png"><img alt="../_images/phylogeny-vocal-learning-comparison-drawn.png" src="../_images/phylogeny-vocal-learning-comparison-drawn.png" style="width: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 13 </span><span class="caption-text">Character mapping of vocal learning behavior on the phylogeny of <em>Neoaves</em> from Brenowitz (1991)<a class="footnote-reference brackets" href="#brenowitz-1991" id="id8">5</a> in <strong>A</strong> and <strong>B</strong> and off of Suh et al (2011)<a class="footnote-reference brackets" href="#suh-2011" id="id9">6</a> in <strong>C</strong>. Yellow: vocal learners, Blue: vocal non-learners.</span><a class="headerlink" href="#id15" title="Permalink to this image">¶</a></p>
</div>
<p><em><strong>Q12: Use the phylogeny in Figure 13A to analyze (using parsimony) the two alternative ancestral state reconstruction hypotheses for the LCA of all song learners (arrow). Make sure to report your quantifications as well as your conclusion.</strong></em></p>
<p><em><strong>Q13: Does “simplifying” the phylogeny in Figure 13B change the results compared to Figure 13A?</strong></em></p>
<p><em><strong>Q14: Use the phylogeny in Figure 13C to analyze (using parsimony) the two alternate ancestral state reconstruction hypotheses for the LCA of all song learners.</strong></em></p>
<p><em><strong>Q15: How does your understanding of vocal learning behavior change based on the two phylogenetic hypotheses (Brenowitz versus Suh)?</strong></em></p>
<p>⏹️ STOP here for today</p>
<div class="admonition-complete-this-section-for-homework-and-bring-your-responses-to-next-class-in-a-format-that-you-can-turn-in-written-on-paper-printed-or-emailed-to-me-directly-are-all-fine-they-will-be-graded-cr-u admonition" id="phylogeny-burrowing-assignment">
<p class="admonition-title">Complete this section for homework and bring your responses to next class in a format that you can turn in (written on paper, printed, or emailed to me directly are all fine). They will be graded CR/U. </p>
<ol class="simple">
<li><p>Examine Figure 2 from <a class="reference external" href="https://doi.org/10.1016/j.anbehav.2008.10.031">Weber and Hoekstra (2009). The evolution of burrowing behaviour in deer mice (genus Peromyscus)</a><a class="footnote-reference brackets" href="#hu-hoekstra-review" id="id10">8</a>. Perform an ancestral state reconstruction for burrowing behavior among these species by examining alternate hypotheses for the LCA and analyzing the results using an argument of parsimony. (All of the information you need should be contained in Figure 2… note that categorization can help reduce the workload of the analysis).</p></li>
</ol>
<p><em>2. Optional</em>: Using the concepts and tools you learned today, research the phylogeny of a species/behavior you are interested in. What are the alternatives to the behavior? How much the the phylogeny do you need to examine interesting convergence and/or divergence for the behavior? Bring your analyis to class.</p>
</div>
</div>
</div>
<div class="section" id="additional-resources">
<h2>Additional Resources<a class="headerlink" href="#additional-resources" title="Permalink to this headline">¶</a></h2>
<ul class="simple">
<li><p><a class="reference external" href="https://www.youtube.com/watch?v=ULzJgG5IBAo&amp;list=PLfdiT8Klm_YPa0lNVa9ygwAjy_1Lpz9_S&amp;index=11">Common Descent Podcast, Episode 10: The Tree of Life</a></p></li>
<li><p>Data for the turtle phylogeny was collected by Lindsay Taylor as part of their <a class="reference external" href="https://taylor-lindsay.github.io/phylogenetics/">example jupyter notebooks implementation of BioPython</a></p></li>
</ul>
<hr class="footnotes docutils" />
<dl class="footnote brackets">
<dt class="label" id="genbank"><span class="brackets"><a class="fn-backref" href="#id1">1</a></span></dt>
<dd><p>If you want to use this resource directly, you can refer to the <a class="reference external" href="https://www.ncbi.nlm.nih.gov/books/NBK44863/">FAQ</a> to get you started.</p>
</dd>
<dt class="label" id="yang-2014"><span class="brackets"><a class="fn-backref" href="#id2">2</a></span></dt>
<dd><p>Yang, L., Tan, Z., Wang, D. et al. <a class="reference external" href="https://doi.org/10.1038/srep04089">Species identification through mitochondrial rRNA genetic analysis</a>. Sci Rep 4, 4089 (2014).</p>
</dd>
<dt class="label" id="colonial"><span class="brackets"><a class="fn-backref" href="#id3">3</a></span></dt>
<dd><p>“Used loosely to describe any group of organisms living together or in close proximity to each other - for example nesting shorebirds that live in large colonies. More specifically refers to a group of organisms in which members act as specialized subunits (a continuous, modular society) - as in clonal organisms.” - <a class="reference external" href="https://animaldiversity.org/accounts/Chelonia_mydas/#behavior">ADW</a>”</p>
</dd>
<dt class="label" id="dna-hybridization"><span class="brackets"><a class="fn-backref" href="#id4">4</a></span></dt>
<dd><p>(from <a class="reference external" href="https://en.wikipedia.org/wiki/DNA%E2%80%93DNA_hybridization">wiki</a>) The DNA of one organism is labelled, then mixed with the unlabelled DNA to be compared against. The mixture is incubated to allow DNA strands to dissociate and then cooled to form renewed hybrid double-stranded DNA. Hybridized sequences with a high degree of similarity will bind more firmly, and require more energy to separate them: i.e. they separate when heated at a higher temperature than dissimilar sequences, a process known as “DNA melting”. To assess the melting profile of the hybridized DNA, the double-stranded DNA is bound to a column and the mixture is heated in small steps. At each step, the column is washed; sequences that melt become single-stranded and wash off the column. The temperatures at which labelled DNA comes off the column reflects the amount of similarity between sequences (and the self-hybridization sample serves as a control). These results are combined to determine the degree of genetic similarity between organisms. One method was introduced for hybridizing large numbers of DNA samples against large numbers of DNA probes on a single membrane. These samples would have to be separated in their own lanes inside the membranes and then the membrane would have to be rotated to a different angle where it would result in simultaneous hybridization with many different DNA probes.</p>
</dd>
<dt class="label" id="brenowitz-1991"><span class="brackets">5</span><span class="fn-backref">(<a href="#id5">1</a>,<a href="#id8">2</a>)</span></dt>
<dd><p><a class="reference external" href="https://doi.org/10.1016/1044-5765%2891%2990030-R">Eliot A. Brenowitz (1991) <em>Evolution of the vocal control system in the avian brain</em>. Seminars in Neuroscience; 3(5).</a></p>
</dd>
<dt class="label" id="suh-2011"><span class="brackets">6</span><span class="fn-backref">(<a href="#id6">1</a>,<a href="#id9">2</a>)</span></dt>
<dd><p><a class="reference external" href="https://doi.org/10.1038/ncomms1448">Suh, A., Paus, M., Kiefmann, M. et al. Mesozoic retroposons reveal parrots as the closest living relatives of passerine birds. Nat Commun 2, 443 (2011).</a></p>
</dd>
<dt class="label" id="retroposons"><span class="brackets"><a class="fn-backref" href="#id7">7</a></span></dt>
<dd><p>(from <a class="reference external" href="https://doi.org/10.1038/ncomms1448">Suh et al 2011</a>) Retroposons, jumping genetic elements that copy via RNA intermediates and insert nearly randomly anywhere in the genome (although some biases of insertion and retention have been proposed15), provide (by inheritance) virtually homoplasy-free evidence of relatedness that is detectable for more than 100 million years. Because parallel insertions or exact excisions are highly unlikely, presence/absence patterns of retroposons at orthologous genomic loci are powerful, clear-cut phylogenetic markers capable of resolving long-standing uncertainties.</p>
</dd>
<dt class="label" id="hu-hoekstra-review"><span class="brackets"><a class="fn-backref" href="#id10">8</a></span></dt>
<dd><p>Figure 1 from <a class="reference external" href="https://doi.org/10.1016/j.semcdb.2016.08.001">Hu and Hoekstra (2017) Peromyscus burrowing: A model system for behavioral evolution. Journal of Cell and Developmental Biology, 61</a> shows a mapping of both burrowing behavior and habitat on the phylogeny of Peromyscus. You can also start prepping for the next section of the course by thinking about… What correlations do you notice? What do you wonder about the function of burrowing behavior and how that relates to its phylogenetic history?</p>
</dd>
</dl>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./levels-of-analysis"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            
                <!-- Previous / next buttons -->
<div class='prev-next-area'> 
    <a class='left-prev' id="prev-link" href="ultimate.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Ultimate</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="evolution.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Evolution</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            
        </div>
    </div>
    <footer class="footer">
  <p>
    
      By Dr. Krista Perks<br/>
    
        &copy; Copyright 2022.<br/>
  </p>
</footer>
</main>


      </div>
    </div>
  
  <script src="../_static/js/index.be7d3bbb2ef33a8344ce.js"></script>

  </body>
</html>